Ago-Dic-2020/chavez-muñoz-juan-arnoldo/Practica-1/Practica-1.py

```diff
@@ -0,0 +1,15 @@
+class Matrix:
+    matrix_string =[]
```

Comment:
No necesitas declarar el atributo aquí, ya que esto sería un atributo de clase y no de la instancia. Checa el siguiente artículo: https://www.digitalocean.com/community/tutorials/understanding-class-and-instance-variables-in-python-3